src/components/ClusterOverview/Details/Details.js

```diff
@@ -64,10 +64,10 @@ DetailsBody.propTypes = {
   LoadingComponent: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
 };
 
-export const Details = ({ items, LoadingComponent }) => (
```

Comment:
IMO it would make more sense to export DetailsBody and create the card itself in storageDetails than to overcustomize it.